Mapsui.Rendering.Skia-PCL/ArrowLineRenderer.cs

```diff
@@ -0,0 +1,21 @@
+﻿using Mapsui.Geometries;
+using Mapsui.Providers;
+using Mapsui.Styles;
+using SkiaSharp;
+
+namespace Mapsui.Rendering.Skia
+{
+    public static class ArrowLineRenderer
+    {
+        public static void Draw(SKCanvas canvas, IViewport viewport, IStyle style, IFeature feature, IGeometry geometry,
+            float opacity)
+        {
+            var arrowLine = (ArrowLine)geometry;
+
+            foreach (var lineString in arrowLine)
+            {
+                LineStringRenderer.Draw(canvas, viewport, style, feature, lineString, opacity);
+            }
```

Comment:
Isn't the MultiLineStringRenderer already sufficient?

Reply:
You must be right